addons/sms_twilio/wizard/sms_twilio_account_manage_views.xml

```diff
@@ -23,17 +23,17 @@
                         required="1"/>
                     <label for="test_number"/>
                     <div class="d-flex align-items-center">
-                        <field name="test_number" placeholder="+1 555-123-4567"/>
+                        <field name="test_number" placeholder="+1 555-123-4567" class="w-auto"/>
                         <button name="action_send_test" string="Send test SMS"
                             icon="fa-send" type="object"
-                            class="btn btn-link w-25 ms-1"/>
+                            class="btn btn-link w-auto ms-1 p-0"/>
                     </div>
                 </group>
                 <group string="Phone Numbers">
                     <button
                         class="btn btn-secondary" colspan="2"
                         type="object" name="action_reload_numbers"
-                        string="Reload Numbers from Twilio" icon="fa-refresh"/>
+                        string="Reload Numbers from Twilio" icon="fa-refresh" invisible="sms_twilio_number_ids"/>
```

Comment:
:thinking: Isn't the point that you can reload if one is added/some change? Even if you already have some

Reply:
Yes we don't agree on that spec, button should always be available, otherwise you have no way to update with new numbers.